vendor/engines/secure_rooms/app/views/secure_rooms/shared/_tabnav_secure_room.html.haml

```diff
@@ -0,0 +1,2 @@
+- if @product.is_a?(SecureRoom)
```

Comment:
Probably not a big deal for now, but to future-proof this, `@product.respond_to?(:card_readers)` might be better.